source/reference/mongosync.txt

```diff
@@ -334,6 +334,14 @@ To sync a source cluster to multiple destination clusters, use one
 information, see :ref:`Multiple Clusters Limitations
 <multiple-clusters-limitations>`. 
 
+Data Definition Language (DDL) Operations
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+DDL operations (operations that act on collections or databases) during
```

Comment:
[suggestion]

Consider starting this sentence with a verb. Perhaps: "Using DDL operations ...".